server.go

```diff
@@ -490,7 +493,7 @@ func (s *Server) Get(def *Def, r io.Reader, w io.Writer) error {
 
 		// Exists. Write it out and go to the next one.
 		if err == nil {
-			fmt.Fprintln(w, alias)
+			fmt.Fprintln(w, "0", alias)
```

Comment:
This should be "1" since a match exists. On 502, the print should be "0" since no match was found.